modules/claims_api/app/controllers/claims_api/v2/veterans/claims_controller.rb

```diff
@@ -149,7 +149,12 @@ def build_claim_structure(data:, lighthouse_id:, upstream_id:)
             status: detect_status(data),
             submitter_application_code: data[:submtr_applcn_type_cd],
             submitter_role_code: data[:submtr_role_type_cd],
-            '5103_waiver_submitted'.to_sym => map_yes_no_to_boolean('filed5103_waiver_ind', data[:filed5103_waiver_ind])
+            '5103_waiver_submitted'.to_sym => map_yes_no_to_boolean('filed5103_waiver_ind',
+                                                                    data[:filed5103_waiver_ind]),
+            jurisdiction: data[:regional_office_jrsdctn],
+            maxEstClaimDate: data[:maxEstClaimDate],
+            minEstClaimDate: data[:minEstClaimDate],
+            tempJurisdiction: data[:temp_regional_office_jrsdctn]
```

Comment:
Total Nit-Picky thing I know, but can we move these attributes up into the alphabetical listing with the others?